cross/dtv-scan-tables/Makefile

```
@@ -0,0 +1,23 @@
+PKG_NAME = dtv-scan-tables
+PKG_VERS = na
+PKG_EXT = tar.gz
+PKG_DIST_NAME = master.$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/crazycat69/dtv-scan-tables/archive
+PKG_DIST_FILE = $(PKG_NAME).$(PKG_EXT)
+PKG_DIR = $(PKG_NAME)-master
+
+DEPENDS =
+
+CONFIGURE_TARGET = nop
+COMPILE_TARGET = nop
+INSTALL_TARGET = dtv_scan_tables_install
+
+HOMEPAGE = https://github.com/crazycat69/dtv-scan-tables
+COMMENT  = Digital TV scan tables
+LICENSE  = N/A
+
+include ../../mk/spksrc.cross-cc.mk
```

you can omit `CONFIGURE_TARGET = nop` and `COMPILE_TARGET = nop` when you use 
```suggestion
include ../../mk/spksrc.install-resources.mk
```